portal/server/helpers/constructPayload.js

```diff
@@ -5,16 +5,24 @@
  * @param {Object} body Request body (req.body)
  * @param {Array} properties Interested properties to be added into payload
  * @param {Boolean} csrf Include CSRF token, defaulted to `true`
+ * @param {Boolean} propertyEmpty List of property names that can't have empty value as they will have object value later, defaulted to True
```

Comment:
please re-word to `Whether property can be empty in a payload` and rename the variable to `canPropertyEmpty`.

Reply:
Changed to `'Whether property can be empty in a payload' `and renamed variable to `canPropertyBeEmpty`